zkevm-circuits/src/evm_circuit/execution.rs

```diff
@@ -246,7 +247,7 @@ pub(crate) struct ExecutionConfig<F> {
     // EVM Circuit selector, which enables all usable rows.  The rows where this selector is
     // disabled won't verify any constraint (they can be unused rows or rows with blinding
     // factors).
-    q_usable: Selector,
+    q_usable: Column<Fixed>,
```

Comment:
if we still use selectors, halo2 MockProver will report err for cross region gates, which are needed for parallel assignment